lib/iex/lib/iex/autocomplete.ex

```diff
@@ -225,8 +225,10 @@ defmodule IEx.Autocomplete do
 
     for mod <- match_modules(base, module === Elixir),
         parts = String.split(mod, "."),
-        depth <= length(parts) do
-      %{kind: :module, type: :elixir, name: Enum.at(parts, depth - 1)}
+        depth <= length(parts),
+        name = Enum.at(parts, depth - 1),
+        not String.starts_with?(inspect(String.to_atom("Elixir." <> name)), ":") do
```

Comment:
I think we can avoid `inspect(String.to_atom` transformations, and also concatenation by doing:
```elixir
for mod <- match_modules(base, module === Elixir),
    Macro.classify_identifier(mod) == :alias
    parts = String.split(mod, "."),
    depth <= length(parts) do
  %{kind: :module, type: :elixir, name: Enum.at(parts, depth - 1)}
end
```